server/build.gradle

```diff
@@ -141,9 +141,11 @@ sourceSets.main.compiledBy(generateModulesList, generatePluginsList)
 if (BuildParams.isSnapshotBuild() == false) {
     tasks.named("test").configure {
         systemProperty 'es.index_mode_feature_flag_registered', 'true'
+        systemProperty 'es.failure_store_feature_flag_enabled', 'true'
```

Comment:
Is there a way to enable this feature flag an ESS project or specific projects? Would be convenient for testing.